backend/main.go

```diff
@@ -60,6 +59,20 @@ func main() {
 	}
 	log.Println("Finished inserting postgres rules.")
 
+	// For demo environment, add account to kick of cartography.
+	if os.Getenv("MODE") == constants.DemoEnvModeStr {
+		accountError := handlers.ManualAddAccountDetails(postgresDb, models.AccountDetails{
+			AccountName:       "ZeusCloudDemo",
+			ConnectionMethod:  "profile",
+			Profile:           "default",
+			LastScanCompleted: nil,
+		})
+		if accountError.ErrorCode != 200 || accountError.Error != "" {
+			log.Printf("Adding account error %+v", accountError)
+			return
```

Comment:
don't return - just log and continue. (possible for account to already be in db in which case adding more than 1 account is not allowed)